src/scheduler/fifo.c

```diff
@@ -2442,31 +2443,34 @@ sched_settings_frm_svr(struct batch_status *status)
 				free(attribs);
 				if (err) {
 					snprintf(log_buffer, sizeof(log_buffer), "Failed to update scheduler comment %s at the server", comment);
-					log_err(-1, __func__, log_buffer);
+					schdlog(PBSEVENT_ERROR, PBS_EVENTCLASS_SCHED, LOG_ERR, __func__, log_buffer);
 				}
-				log_dir = tmp_log_dir;
-				validation_failed = 1;
+				if (tmp_log_dir != NULL)
+					free(tmp_log_dir);
+				if (tmp_priv_dir != NULL)
+					free(tmp_priv_dir);
+				if (tmp_partitions != NULL)
+					free(tmp_partitions);
+				if (tmp_comment != NULL)
+					free(tmp_comment);
```

Comment:
Same as above about checking against NULL (and all other instances of this)

Reply:
Done.